readme.md

```diff
@@ -26,7 +35,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+By doing requests to the client asynchronously
```

Comment:
Which HTTP method would be used? Also, requests aren't made *to* the client; they're made *by* the client *to* the server.